UserInterface/ViewModels/MainWindowViewModel.cs

```diff
@@ -174,20 +174,23 @@ private void OnCurrentViewModelChanged()
 
     private void ChangeToBig(ResourceDictionary resourceDict)
     {
-        ((Style)resourceDict["Text"]).Setters.Add(new Setter(TextBlock.FontSizeProperty, 40.0));
-        ((Style)resourceDict["SubHeader"]).Setters.Add(new Setter(TextBlock.FontSizeProperty, 48.0));
-        ((Style)resourceDict["Header"]).Setters.Add(new Setter(TextBlock.FontSizeProperty, 56.0));
+        UpdateResource(resourceDict, "Text", 40.0);
```

Comment:
Nice die updateResource! Wellicht dat van de fontsizes nog een variable gemaakt kan worden? Of twee toevoegingen aan de resourceDictrionary? Absoluut geen must though

Reply:
Eens, dat zal wel mooier zijn, moeten even kijken of we hier iets voor maken.